File: KeybondAndroid/app/src/main/java/com/keybond/android/ime/Theme.kt
package com.keybond.android.ime

import androidx.compose.ui.graphics.Color

/** Colors and metrics tuned to match the stock iOS/Android dark keyboard look. */
object KeyboardColors {
    val background = Color(0xFF1B1B1D)
    val letterKey = Color(0xFF4A4A4E)
    val letterKeyPressed = Color(0xFF6B6B70)
    val specialKey = Color(0xFF29292D)
    val specialKeyPressed = Color(0xFF4D4D51)
    val specialKeyActive = Color.White
    val keyText = Color.White
    val keyTextOnActive = Color.Black
    val suggestionDivider = Color.White.copy(alpha = 0.25f)
    val placeholderText = Color.White.copy(alpha = 0.55f)
}

object KeyboardMetrics {
    const val KEY_CORNER_RADIUS_DP = 5
    const val ROW_SPACING_DP = 10
    const val KEY_SPACING_DP = 6
    const val KEY_HEIGHT_DP = 42
}

File: KeybondKeyboard/Theme.swift
import SwiftUI

/// Colors and metrics tuned to match the stock iOS dark keyboard.
enum Theme {
    static let background = Color(red: 0.106, green: 0.106, blue: 0.114) // iOS keyboard dark background
    static let letterKey = Color(red: 0.290, green: 0.290, blue: 0.306) // light gray key
    static let letterKeyPressed = Color(red: 0.42, green: 0.42, blue: 0.44)
    static let specialKey = Color(red: 0.161, green: 0.161, blue: 0.176) // darker key (shift, backspace, 123)
    static let specialKeyPressed = Color(red: 0.30, green: 0.30, blue: 0.32)
    static let specialKeyActive = Color.white // shift/caps active state
    static let keyText = Color.white
    static let keyTextOnActive = Color.black
    static let suggestionBarBackground = Color(red: 0.106, green: 0.106, blue: 0.114)
    static let suggestionDivider = Color.white.opacity(0.25)
    static let placeholderText = Color.white.opacity(0.55)

    static let keyCornerRadius: CGFloat = 5
    static let rowSpacing: CGFloat = 10
    static let keySpacing: CGFloat = 6
}
